src/pages/index.tsx

```diff
@@ -155,13 +154,13 @@ const Home: React.FunctionComponent<HomeProps & NextPage> = ({
             {status === 'authenticated' && !isValidating && (
                 <RecommendedSection
                     tags={recommendedTags}
+                    followedTags={followedTags}
                     content={content}
                     showFooter
                     onTagsSaved={(
                         followedTags: Tag[],
                         emailPreference: boolean
                     ) => {
-                        // TODO: Replace with call to user preferences endpoint
                         submitPersonalizationSelections({
```

Comment:
if you want/have the capacity, you can add the below so that the Toasts can appear for success/failure when saving the recommended tags (it's already added on the modal callbacks so no need to add it there). I have an active branch for a couple small fixes so I can add it there if needed.

```
        const { error } = await submitPersonalizationSelections({
                 followedTags,
                 emailPreference,
        });

        if (!error) {
            setNotification({
                message: 'Successfully saved your preferences',
                variant: 'SUCCESS',
            });
        } else {
            setNotification({
                message:
                    'Your request could not be completed at this time. Please try again.',
                variant: 'WARN',
            });
        }
```